options/locale/locale_en-US.ini

```diff
@@ -1029,6 +1029,8 @@ issues.owner = Owner
 issues.re_request_review=Re-request review
 issues.remove_request_review=Remove review request
 issues.remove_request_review_block=Can't remove review request
+issues.dismiss_review_btn = Dismiss review
+issues.dismiss_review_confirm = Dismiss review confirm
```

Comment:
Not sure of this phrasing. Usually our modals look like:

## Dismiss Review

Are you sure you want to dismiss this review?

**OK** Cancel